File: exercicios/cpp/Instruções Condicionais/Q19.cpp
#include <iostream>
#include <clocale>

using namespace std;

int main() { 
	setlocale(LC_ALL, "Portuguese");
	float t1, t2, t3; 
	string n1, n2, n3; 
	cout<<"Informe o nome do primeiro atleta:";
	cin>>n1;
	cout<<"Informe o tempo do primeiro atleta:";
	cin>>t1;
	cout<<"Informe o nome do segundo atleta:";
	cin>>n2;
	cout<<"Informe o tempo do segundo atleta:";
	cin>>t2;
	cout<<"Informe o nome do terceiro atleta:";
	cin>>n3;
	cout<<"Informe o tempo do terceiro atleta:";
	cin>>t3;			
	
	if (t1 < t2) {
		if (t1 < t3) {
			cout<<n1<<" Campeão"<<endl; 
			if (t2 < t3) { 
				cout<<n2<<" Vice-campeão"<<endl; 
				cout<<n3<<" Terceiro Lugar"<<endl; 
			} else { 
				cout<<n3<<" Vice-campeão"<<endl; 
				cout<<n2<<" Terceiro Lugar"<<endl; 
			}
		} else {
			cout<<n3<<" Campeão"; 
			if (t1 < t2) { 
				cout<<n1<<" Vice-campeão"<<endl; 
				cout<<n2<<" Terceiro Lugar"<<endl; 
			} else { 
				cout<<n2<<" Vice-campeão"<<endl; 
				cout<<n1<<" Terceiro Lugar"<<endl; } 
			} 
	} else { //t2 ganha do t1 
		if (t2 < t3) { 
			cout<<n2<<" Campeão"<<endl; 
			if (t1 < t3) { 
				cout<<n1<<" Vice-campeão"<<endl; 
				cout<<n3<<"Terceiro Lugar"<<endl; 
			} else { 
				cout<<n3<<" Vice-Campeão"<<endl; 
				cout<<n1<<" Terceiro Lugar"<<endl; 
			} 
		} else { 
			cout<<n3 <<" Campeão"<<endl; 
			if (t1 < t2) { 
				cout<<n1<<" Vice-Campeão"<<endl; 
				cout<<n2<<" Terceiro Lugar"<<endl; 
			} else { 
				cout<<n2<<" Vice-Campeão"<<endl; 
				cout<<n1<<" Terceiro Lugar"<<endl; 
			} 
		} 
	} 
	return 0;
}

File: exercicios/python/Instruções Condicionais/Q19.py
# 19 - Corrida de atletas sem usar array

nome1 = input("Nome do atleta 1: ")
tempo1 = float(input("Tempo do atleta 1: "))

nome2 = input("Nome do atleta 2: ")
tempo2 = float(input("Tempo do atleta 2: "))

nome3 = input("Nome do atleta 3: ")
tempo3 = float(input("Tempo do atleta 3: "))

# Verifica quem foi o campeão, vice e terceiro
if tempo1 < tempo2 and tempo1 < tempo3:
    campeao = nome1
    if tempo2 < tempo3:
        vice = nome2
        terceiro = nome3
    else:
        vice = nome3
        terceiro = nome2
elif tempo2 < tempo1 and tempo2 < tempo3:
    campeao = nome2
    if tempo1 < tempo3:
        vice = nome1
        terceiro = nome3
    else:
        vice = nome3
        terceiro = nome1
else:
    campeao = nome3
    if tempo1 < tempo2:
        vice = nome1
        terceiro = nome2
    else:
        vice = nome2
        terceiro = nome1

print("Campeão:", campeao)
print("Vice-campeão:", vice)
print("3º lugar:", terceiro)